datalad/distribution/install.py

```diff
@@ -299,6 +299,40 @@ def __call__(
             recursion_limit, git_opts, git_clone_opts, annex_opts,
             annex_init_opts)
 
+        # handle calls with multiple paths first:
+        already_done = []
+        if path:
+            if isinstance(path, list):
+                _cnt = len(path)
+                if _cnt == 0 :
```

Comment:
I would have done it as 
```python
if not path:
   return
else:
  if len(path) > 1 and source:
     raise 
  for p in path:
   ...
```

so yes -- for any cmdline invocation (would be a list) it would call itself again. BUT that would make it uniform, so would be more likely we would catch a bug e.g. if some option is not passed in